crun.1

```diff
@@ -648,16 +648,21 @@ It is an experimental feature.
 
 .PP
 If specified, run the specified handler for execing the container.
-The only supported value is \fB\fCkrun\fR\&.  When \fB\fCkrun\fR is specified, the
-\fB\fClibkrun.so\fR shared object is loaded and it is used to launch the
-container using libkrun.
+The only supported values are \fB\fCkrun\fR and \fB\fCwasm\fR\&.
 
-.SH \fB\fCrun.oci.handler=wasm\fR
-.PP
-If specified, run the wasm handler for container.
-Allows running wasm workload natively. Accepts a \fB\fC\&.wasm\fR binary as input
-and if \fB\fC\&.wat\fR is provided it will automatically compiled into a wasm module.
-Stdout of wasm module is relayed back via crun.
+.RS
+.IP \(bu 2
+
+.SS krunWhen \fB\fCkrun\fR is specified, the \fB\fClibkrun.so\fR shared object is loaded
```

Comment:
After `.SS krun` and before `When` is missing a new line or white space character.

Likewise for `.SS wasmIf` below.

Reply:
![crun-krun](https://user-images.githubusercontent.com/4992947/197326094-f9db1b52-061d-48fa-b9f0-f2606fa26700.png)

On my computer the bullets are literally in the middle of nowhere.

Reply:
Hmm weird, I used `md2man` let me check this.

Reply:
Following PR should fix it: https://github.com/containers/crun/pull/1038